mkosi/installer/moss.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from collections.abc import Sequence
+
+from mkosi.bubblewrap import apivfs_cmd, bwrap
+from mkosi.context import Context
+from mkosi.types import PathString
+from mkosi.util import sort_packages
+
+
+def moss_cmd(context: Context) -> list[PathString]:
+    cmdline: list[PathString] = [
+        "moss",
+        "--directory",
+        f"{context.root}",
+        "--yes-all",
+    ]
+
+    return cmdline
+
+
+def setup_moss(
+    context: Context,
+) -> None:
+    (context.pkgmngr / "etc/moss/repo.d").mkdir(exist_ok=True, parents=True)
+    bwrap(context,
+        moss_cmd(context)
+        + ["repo", "add", "volatile", "https://dev.serpentos.com/volatile/x86_64/stone.index"],
+        network=True, env=context.config.environment,
+    )
+
+def invoke_moss(
+    context: Context,
+    operation: str,
+    packages: Sequence[str] = (),
+    apivfs: bool = False,
+) -> None:
+    cmd = apivfs_cmd(context.root) if apivfs else []
+    bwrap(context, cmd + moss_cmd(context) + [operation] + [*sort_packages(packages)],
+        network=True, env=context.config.environment,
+    )
```

Comment:
formatting and no need to unpack the list of packages and pack it up again.
```suggestion
    bwrap(
        context, 
        cmd + moss_cmd(context) + [operation] + sort_packages(packages),
        network=True, 
        env=context.config.environment,
    )
```